addons/stock/models/res_company.py

```diff
@@ -130,15 +129,15 @@ def create_missing_transit_location(self):
     def create_missing_inventory_loss_location(self):
         company_ids  = self.env['res.company'].search([])
         inventory_loss_product_template_field = self.env['ir.model.fields'].search([('model','=','product.template'),('name','=','property_stock_inventory')])
```

Comment:
There is a cache for that, use it and save queries:
```suggestion
        inventory_loss_product_template_field = self.env['ir.model.fields']._get('product.template', 'property_stock_inventory')
```